.travis.yml

```diff
@@ -26,8 +26,9 @@ env:
     - ROS_DISTRO=indigo NOT_TEST_BUILD='true' NOT_TEST_INSTALL='true' POST_PROCESS='I_am_supposed_to_fail'
     - ROS_DISTRO=indigo CATKIN_PARALLEL_JOBS='-p1' ROS_PARALLEL_JOBS='-j1'  # Intend build on low-power platform
     # - env: ROS_DISTRO=indigo PRERELEASE=true  ## Comment out because this is meaningless for always failing without prerelease testable contents in industrial_ci.
-    - ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_REPONAME=std_msgs PRERELEASE_DOWNSTREAM_DEPTH=0
-    - ROS_DISTRO=kinetic PRERELEASE=true PRERELEASE_REPONAME=std_msgs
+    - ROS_DISTRO=indigo PRERELEASE=true PRERELEASE_DOWNSTREAM_DEPTH=1
```

Comment:
This is not possible for now. With this patch, we cannot "fake" the identity anymore, and `industrial_ci` does not have any dowstream depencies.
In a later versions we might add a full rosinstall workspace as underlay, then we can test it again